fastn-core/src/library2022/processor/sqlite.rs

```diff
@@ -33,6 +33,44 @@ pub async fn process(
 ) -> ftd::interpreter::Result<ftd::interpreter::Value> {
     let (headers, query) = get_p1_data("package-data", &value, doc.name)?;
 
+    let use_db = match headers.get_optional_string_by_key("use", doc.name, value.line_number()) {
+        Ok(Some(k)) => Some(k),
+        _ => match headers
+            .get_optional_string_by_key("db", doc.name, value.line_number())
+            .ok()
+        {
+            Some(k) => {
+                println!("Warning: `db` header is deprecated, use `use` instead.");
+                k
+            }
+            None => None,
+        },
+    };
+
+    let sqlite_database_path = match use_db {
+        Some(k) => {
+            let mut db_path = camino::Utf8PathBuf::new().join(k.as_str());
```

Comment:
Can we do this instead? I think this should work.

```suggestion
            let mut db_path = config.root.join(k.as_str());
```

Reply:
This is better. All file access must happen with root as soon we are going to have multiple fastn packages in different root directories.